File: BLEVersion/Models/dbReading.cs
using System;

namespace com.gravicode.neospectralib.Models
{
    public class dbReading : Serializable
    {
        private double[] yReading;
        private double[] xReading;

        public dbReading()
        {
        }

        public double[] getXReading()
        {
            return xReading;
        }

        public double[] getYReading()
        {
            return yReading;
        }

        public void setReading(double[] yReading, double[] xReading)
        {
            this.yReading = yReading;
            this.xReading = xReading;
        }

        public override string ToString()
        {
            return "dbReading{" +
                "yReading=" + string.Join(",", yReading) +
                ", xReading=" + string.Join(",", xReading) +
                '}';
        }
    }
}

File: BLEVersion/JavaVersion/Models/dbReading.java
package com.gravicode.neospectralib.Models;

import java.io.Serializable;
import java.util.Arrays;

/**
 *
 */

public class dbReading implements Serializable {
    private double[] yReading, xReading;

    public dbReading() {
    }

    public double[] getXReading() {
        return xReading;
    }

    public double[] getYReading() {
        return yReading;
    }

    public void setReading(double[] yReading, double[] xReading) {
        this.yReading = yReading;
        this.xReading = xReading;
    }

    @Override
    public String toString() {
        return "dbReading{" +
                "yReading=" + Arrays.toString(yReading) +
                ", xReading=" + Arrays.toString(xReading) +
                '}';
    }
}
